x-pack/plugins/security_solution/public/management/cypress/tasks/alert_details_actions.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const openAlertDetailsView = (): void => {
+  cy.getByTestSubj('expand-event').first().click();
+  cy.getByTestSubj('take-action-dropdown-btn').click();
+};
+
+export const openInvestigateInTimelineView = (): void => {
```

Comment:
same here - maybe name it with the word "first"

Reply:
602dbc64a24124472991020ba6dbfda39408553a